app/migrations/Version20180425145459.php

```diff
@@ -10,24 +10,26 @@ class Version20180425145459 extends AbstractMigration
     public function up(Schema $schema)
     {
         $this->addSql('
-            CREATE TABLE adherent_email_subscription_history (
```

Comment:
This PR is based on this one https://github.com/EnMarche/en-marche.fr/pull/2374/files#diff-646623fd47639d7f596336914941814cR8. 
It seems to be more correct to change migration in the base PR, because if it will be merged first the migration will be executed and these modifications will not be taken into account.